Updated objectListWidget using JSONField as an array (#491)
kitconcept/volto-light-theme: docs/reference/helpers.md

> The successor of Volto's `DragDropList`, using `dnd-kit` library.

Reviewer: ```suggestion
`DndSortableList is the successor of Volto's `DragDropList`, using the `dnd-kit` library.
```

> Given a `items` array of objects, and a children render prop function, it allows you to reorder them using drag and drop.

Reviewer: ```suggestion
Given an array of objects as `items`, and a `children` render prop function, you can drag and drop the items to reorder them.
```
